signalfx-tracing/signalfx-java-tracing/jetty-customtags/run-server.sh

```diff
@@ -6,4 +6,4 @@ export SIGNALFX_SERVICE_NAME=EchoServer-JavaTagDemo
 
 mvn compile exec:exec \
   -Dexec.executable="java" \
-  -Dexec.args="-javaagent:/opt/signalfx-tracing.jar -cp %classpath sf.main.EchoDemo 5000"
+  -Dexec.args="-javaagent:/opt/signalfx-tracing.jar -Dsignalfx.endpoint.url=http://127.0.0.1:8888 -cp %classpath sf.main.EchoDemo 5000"
```

Comment:
Why this change?

Reply:
Git is stupid, that's why (will revert ofc ;) ).